packages/common/src/system/guid.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// @ts-ignore
+import * as uuid from 'uuid-with-v6';
+
+// tslint:disable: no-unsafe-any
+
+export namespace Guid {
+    export function createGuid(lastGuid?: string): string {
```

Comment:
>lastGuid?: string [](start = 31, length = 17)

have a separate function for when u pass lastGuid:
createGuidWithSameLast8Bytes
or createGuidFrom(guid: string)

Reply:
ok